Amplify/Categories/DataStore/Model/Collection/List+Combine.swift

```diff
@@ -1,32 +0,0 @@
-//
-// Copyright 2018-2020 Amazon.com,
-// Inc. or its affiliates. All Rights Reserved.
-//
-// SPDX-License-Identifier: Apache-2.0
-//
-
-import Combine
-
-@available(iOS 13.0, *)
-extension List {
-
-    public typealias LazyListPublisher = AnyPublisher<Elements, DataStoreError>
-
-    /// Lazy load the collection and expose the loaded `Elements` as a Combine `Publisher`.
-    /// This is useful for integrating the `List<ModelType>` with existing Combine code
-    /// and/or SwiftUI.
-    ///
-    /// - Returns: a type-erased Combine publisher
-    public func loadAsPublisher() -> LazyListPublisher {
```

Comment:
this has been moved to List as an open func for DataStoreList to continue supporting this public API